content/pt-br/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Instalando a ferramenta kubeadm
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+<img src="https://raw.githubusercontent.com/kubernetes/kubeadm/master/logos/stacked/color/kubeadm-stacked-color.png" align="right" width="150px">Essa página mostra o processo de instalação do conjunto de ferramentas `kubeadm`. 
+Para mais informações sobre como criar um cluster com o kubeadm após efetuar a instalação, veja a página [Utilizando kubeadm para criar um cluster](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
+
+
+## {{% heading "prerequisites" %}}
+
+* Uma máquina com sistema operacional Linux compatível. O projeto Kubernetes provê instruções para distribuições Linux baseadas em Debian e Red Hat, bem como para distribuições sem um gerenciador de pacotes.
+* 2 GB ou mais de RAM por máquina (menos que isso deixará pouca memória para as suas aplicações).
+* 2 CPUs ou mais.
+* Conexão de rede entre todas as máquinas no cluster. Seja essa pública ou privada.
+* Nome da máquina na rede, endereço MAC e producy_uuid únicos para cada nó. Mais detalhes podem ser lidos [aqui](#veficiar-endereco-mac).
+* Portas específicas abertas nas suas máquinas. Você poderá ler quais são [aqui](#verificar-portas-necessarias).
+* Swap desabilitado. Você *precisa* desabilitar a funcionalidade de swap para que o kubelet funcione de forma correta.
+
+<!-- steps -->
+
+## Verificando se o endereço MAC e o product_uiid são únicos para cada nó {#veficiar-endereco-mac}
+
+* Você pode verificar o endereço MAC da interface de rede utilizando o comando `ip link` ou o comando `ipconfig -a`.
+* O product_uuid pode ser verificado utilizando o comando `sudo cat /sys/class/dmi/I'd/product_uuid`.
```

Comment:
Typo here: `sudo cat /sys/class/dmi/I'd/product_uuid`

Reply:
Great catch!
Must have been the grammar checker hahaha. 
Fixed.